evap/evaluation/models.py

```diff
@@ -147,7 +161,7 @@ class Course(models.Model, metaclass=LocalizeModelBase):
     type = models.CharField(max_length=1024, verbose_name=_("type"))
 
     # bachelor, master, d-school course
```

Comment:
could you change this to something neutral like "e.g. Bachelor Information Science, Master something else"